brownie/network/account.py

```diff
@@ -944,26 +944,24 @@ def _apply_fee_to_tx(
 ):
     tx = tx.copy()
 
-    if gas_price is not None and (max_fee or priority_fee):
-        raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+    if gas_price is not None:
+        if max_fee or priority_fee:
+            raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+        tx["gasPrice"] = web3.toHex(gas_price)
+        return tx
 
-    elif max_fee is not None and priority_fee is not None:
-        if priority_fee > max_fee:
-            raise InvalidTransaction("priority_fee must not exceed max_fee")
+    if priority_fee is None:
+        raise InvalidTransaction("priority_fee must be defined")
 
     # no max_fee specified, infer from base_fee
-    elif max_fee is None and priority_fee is not None:
+    if max_fee is None:
         base_fee = Chain().base_fee
         max_fee = base_fee * 2 + priority_fee
```

Comment:
2 gwei is a constant [from here](https://notes.ethereum.org/@barnabe/rk5ue1WF_), but in reality we want to predict an optimal priority fee. In the future we can use [`eth_feeHistory` api](https://github.com/zsfelfoldi/feehistory) for that, but it's not implemented in many clients at the moment.

Reply:
Make an issue and come back when the clients are ready, I guess.